general/development/process/triage.md

```diff
@@ -0,0 +1,233 @@
+```
+
+```txt title="Request to correct translation"
+Thanks for reporting this.
+
+The problem that you are describing refers to the translation in another language. Language string problems can be corrected by [a translation|https://docs.moodle.org/dev/Contributing_a_translation](Contributing) or by contacting the language pack maintainer as listed in the [credits|http://lang.moodle.org/local/amos/credits.php](Translation). I'm going to close this issue because Moodle does not include translations in the standard download package.
+```
+
+```txt title="Correction to English language strings"
+Thanks for reporting this.
+
+Please note that English language string typo fixes and suggested improvements can be [to the English (fixes) (en_fix) language pack|https://docs.moodle.org/dev/Improving_English_language_strings](contributed)
+```
+
+```txt title="Bug in contributed plugin (plugin can be identified)"
+You have created an issue under MDL project in the tracker. However the problem that you are describing refers to contributed plugin xxx.
+
+I have moved your issue in the appropriate component of the CONTRIB project in tracker. I will also recommend you to leave a comment on the plugin page in the [plugins directory|https://moodle.org/plugins/](Moodle)
+```
+
+```txt title="Triaging a bug report"
+Thanks for reporting this issue.
+
+You can help us resolve it as quickly as possible by:
+
+- Adding further information in a comment or attaching a screenshot if requested by a developer investigating the issue.
+- Posting the issue number in a moodle.org forum discussion about it.
+- If you are able to provide a patch or links to your Git repository branch, please add a patch label so we will spot it.
+```
+
+```txt title="Triaging an improvement or new feature"
+Thanks for reporting this.
+
+Adding more detail to your suggestion will make it easier to work on.
+
+If you can propose a code solution, that will help others who may have the same need and will increase the chance of this improvement/feature coming about sooner. If you are able to provide a patch or links to your Git repository branch, please add a patch label so we will spot it. Otherwise, how about posting in a forum on moodle.org and encouraging people to vote, comment and/or come up with a patch.
+
+For more ways to maximize the chance of your idea being implemented, see the guide [new feature ideas|https://docs.moodle.org/dev/New_feature_ideas].
+```
+
+## Creating a filter and gadget for triaging
+
+If you are a component lead you are responsible for triaging issues that involve your component. A good way to monitor newly created issues is to create a filter in the Tracker and add a gadget on your dashboard to show the results of the filter.
+
+### Adding a filter
```

Comment:
Maybe we should move these to the https://moodle.github.io/devdocs/general/development/tracker/tips page  and link there instead? Maybe not. Not sure.

Reply:
You're right! I think these sections (Creating a filter and Adding a filter in a gadget to your dashboard) fit better in the "Tracker Tips&Tricks" page so I've moved these changes there, improving the pre-existing information and adding a reference from the Bug triage page ;-)